lib/system/assign.nim

```diff
@@ -40,19 +40,20 @@ proc genericAssignAux(dest, src: pointer, n: ptr TNimNode,
   #  echo "ugh memory corruption! ", n.kind
   #  quit 1
 
-template deepSeqAssignImpl(operation, additionalArg) {.dirty.} =
-  var d = cast[ptr NimSeqV2Reimpl](dest)
-  var s = cast[ptr NimSeqV2Reimpl](src)
-  d.len = s.len
-  let elem = mt.base
-  d.p = cast[ptr NimSeqPayloadReimpl](newSeqPayload(s.len, elem.size, elem.align))
+when defined(nimSeqsV2):
```

Comment:
In a future PR should we simply permanently switch to seqv2 and be done with v1?

Reply:
I have no opinion on nimSeqsV2, but I do have an opinion on nim-gdb. And that part was written with current seq implementation. I personally think that gdb support is more important than a new seq version. I also wrote a test for it
in `tests/untestable/gdb/gdb_pretty_printer_test_run.sh`. It is in `untestable` because I wasn't able to integrate it into `testament` at that point in time. That should be done though. Currently it is manual testing only.